src/main/java/org/example/service/UserService.java

```diff
@@ -0,0 +1,78 @@
+package org.example.service;
+
+import org.example.dto.UserDTO;
+import org.example.entity.User;
+import org.example.repository.RoleRepository;
+import org.example.repository.UserRepository;
+import org.example.util.TransformerDTOUser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+
+@Service
+public class UserService {
+
+    @Autowired
+    protected UserRepository userRepository;
+
+    @Autowired
+    protected TransformerDTOUser transformerDTOUser;
+
+    @Autowired
+    protected RoleRepository roleRepository;
+
+    @Autowired
+    protected PasswordEncoder passwordEncoder;
+
+    public User saveUser(UserDTO userDTO) {
+        int id = 2;
```

Comment:
It's a magic number!
Let's move to the class-level constant and name it like USER_ROLE